playbooks/templates/.github/workflows/tft.yml

```diff
@@ -22,7 +22,7 @@ jobs:
     if: |
       github.event.issue.pull_request
       && contains(github.event.comment.body, '[citest]')
-      && (contains(fromJson('["OWNER", "MEMBER", "COLLABORATOR", "CONTRIBUTOR"]'), github.event.comment.author_association)
+      && (contains(fromJson('["OWNER", "MEMBER"]'), github.event.comment.author_association)
```

Comment:
I think "COLLABORATOR" is fine and should stay -- it's an explicit list which is easy to control. 

This also needs to be applied to tft_citest_bad